src/Pages/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,50 @@
+import { html, css, LitElement } from 'lit-element';
+import defaultStyles from '../../FeApp.style.js';
+import { FeServices } from '../../FeServices.js';
+
```

Comment:
you didnt import the login component here, it will fail with application level

Reply:
and notification import also

Reply:
I will import them and update